apps/backend/src/types/collections.ts

```diff
@@ -1,4 +1,12 @@
 import type { Authentication, User } from "@/payload-types"
 
+/*
+ * The Authentication Data Service typings
+ */
 export type CreateAuthenticationData = Omit<Authentication, "id" | "createdAt" | "updatedAt">
+
+/*
+ * The User Data Service typings
+ */
 export type CreateUserData = Omit<User, "id" | "createdAt" | "updatedAt">
+export type EditUserData = Partial<CreateUserData>
```

Comment:
Shouldn't JSDoc look like this?
Also, if you are adding JSDoc to the first 2 types, why not for the `EditUserData`?
```suggestion
import type { Authentication, User } from "@/payload-types"

/*
 * Type for creating authentication data without system-generated fields
 */
export type CreateAuthenticationData = Omit<Authentication, "id" | "createdAt" | "updatedAt">

/*
 * Type for creating user data without system-generated fields
 */
export type CreateUserData = Omit<User, "id" | "createdAt" | "updatedAt">

/*
 * Type for partially editing user data
 */
export type EditUserData = Partial<CreateUserData>
```

Reply:
Sure that makes sense! 

The implementation in this PR reduces the number of comments, it was used to group the typings rather than provide JSDoc as it was quite self-explanatory based on the variable name.